rosa_planning/rosa-understanding-terraform.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+[id="rosa-understanding-terraform"]
+= Preparing Terraform to install ROSA clusters
+:context: rosa-understanding-terraform
+
+toc::[]
+
+Terraform is an infrastructure-as-code tool that provides a way to configure your resources once and replicate those resources as desired. Terraform accomplishes the creation tasks by using declarative language. You declare what you want the final state of the infrastructure resource to be, and Terraform creates these resources to your specifications.
+
+:FeatureName: Red Hat Cloud Services Terraform Provider 
+include::snippets/technology-preview.adoc[]
+
+include::modules/rosa-sts-terraform-prerequisites.adoc[leveloffset=+1]
+
+[discrete]
+[role="_additional-resources"]
+[id="additional-resources_rosa-terraform-prereq"]
+=== Additional resources
```

Comment:
[For additional resource sections that relate to a module and not an assembly](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#:~:text=Only%20use%20.%20formatting%20(.Additional%20resources)%20in,module%20and%20not%20to%20the%20assembly.%20For%20example%3A), I believe the following formatting should be used:

```suggestion
.Additional resources
```